### PR TITLE
Ignore AMP content.

### PR DESCRIPTION
**Pull request by a-bougouin**
### Purpose/goal of Pull Request (Issue #)
Do nothing for AMP content

### Comments
WOVN widget does not support AMP content, so we should ignore those pages. If we don't ignore them the pages could be added on WOVN.io (e.g. if page is directly open in browser for testing). AMP content can be detected by looking at the HTML tag (https://www.ampproject.org/docs/tutorials/create/basic_markup).

### CODE REVIEW

File: lib/wovnrb.rb [hunk 1]
```diff
@@ -104,7 +103,21 @@ def switch_lang(body, values, url, lang=@store.settings['default_lang'], headers
...
+    # To do so, it looks at the required attributes for the HTML tag:
+    # https://www.ampproject.org/docs/tutorials/create/basic_markup.
+    #
+    # @param {Nokogiri::HTML5::Document} body The HTML body to check.
+    #
+    # @returns {Boolean} True is the HTML body is an AMP, false otherwise.
+    def is_amp_page?(body)
+      html_attributes = body.xpath('//html')[0].try(:attributes) || {}
+
+      !html_attributes['amp'].nil? || !html_attributes["\u26A1"].nil?
```
Comment (hythof) [inline lib/wovnrb.rb]: You can also write `html_attributes['amp'] || html_attributes["\u26A1"]`, that is more simple.

Comment (a-bougouin) [reply lib/wovnrb.rb]: I changed, thanks

File: lib/wovnrb.rb [hunk 2]
```diff
@@ -86,10 +86,9 @@ def switch_lang(body, values, url, lang=@store.settings['default_lang'], headers
         d.encoding = "UTF-8"
 
         # If this page has wovn-ignore in the html tag, don't do anything
-        if ignore_all || !d.xpath('//html[@wovn-ignore]').empty?
+        if ignore_all || !d.xpath('//html[@wovn-ignore]').empty? || is_amp_page?(d)
           ignore_all = true
-          output = d.to_html.gsub(/href="([^"]*)"/) { |m| "href=\"#{URI.decode($1)}\"" }
```
Comment (mrasu) [inline lib/wovnrb.rb]: can we remove this line?

Comment (a-bougouin) [reply lib/wovnrb.rb]: This line is creating output from html that has not been change and is doing URI fixes.
If content should not be changed then it makes sens to return the original content.

This broke no tests so it seems right?

Comment (mrasu) [reply lib/wovnrb.rb]: I agree this is not needed.
This change breaks backward compatibility for not only amp users.
So even if there is no test, we need to check there is no affect for users..

Comment (a-bougouin) [reply lib/wovnrb.rb]: then i revert

**Pull request merged**